File: InternationalWagesManager.Models/WorkConditions.cs
namespace InternationalWagesManager.Models
{
    public class WorkConditions
    {
        public int Id { get; set; }
        public Employee Employee { get; set; }
        public int EmployeeId { get; set; }
        public DateTime Date { get; set; }
        public float PayRate { get; set; }
        public Currency WageCurrency { get; set; }
        public int WageCurrencyId { get; set; }
        public Currency ExpensesCurrency { get; set; }
        public int? ExpensesCurrencyId { get; set; }
        public Currency PayCurrency { get; set; }
        public int? PayCurrencyId { get; set; }
        public decimal Deductions { get; set; }

    }
}

File: AngularClient/src/app/dto/workConditions.ts
import { Currency } from './currency';
import { Employee } from './employee'
export class WorkConditions {
  constructor(public id?: number, public employee?: Employee,
    public employeeId?: number, public date?: Date, public payRate?: number,
    public wageCurrency: Currency = new Currency(), public wageCurrencyId?: number,
    public expensesCurrency: Currency = new Currency(), public expensesCurrencyId?: number,
    public payCurrency: Currency = new Currency(), public payCurrencyId?: number,
    public deductions?: number) {

  }
}
